e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js

```diff
@@ -221,6 +221,41 @@ test.describe('Notebook Network Request Inspection @couchdb', () => {
 
         expect(filterNonFetchRequests(addingNotebookElementsRequests).length).toBeLessThanOrEqual(4);
     });
+
+    test('Search tests', async ({ page }) => {
```

Comment:
Should we put this test in its own test suite since it's not doing any network inspection?

Reply:
The test suite itself is named too narrowly. I've renamed it to be more generic.